_common/GFMAlerts.php

```diff
@@ -56,42 +52,40 @@ protected function blockQuote($line)
                     array(
                         'name' => 'p',
                         'attributes' => array('class' => "markdown-alert-title title-$alertType"),
-                        'rawHtml' => $iconType
+                        'rawHtml' => $alertTitleHtml
                     ),
                 )
             )
         );
-        // dd($block);
         return $block;
     }
 
     protected function blockQuoteContinue($line, array $block) {
+        // If the block didn't start as an alert, fall through to blockquote handler
+        if(!isset($block['isAlert'])) {
+            return parent::blockQuoteContinue($line, $block);
+        }
+
         if(!preg_match('/^\>\s(.*)/', $line['text'], $matches)) {
-            return null;
+            return;
         }
-        
+
         // Recognize a new Alert
         if(preg_match('/^\>\s*\[\!(IMPORTANT|TIP|NOTE|WARNING|CAUTION)\]/i', $line['text'])) {
-            return null;
+            return;
         }
 
         // Handles any interruption
         if(isset($block['interrupted'])) {
-            return $block;
+            return;
```

Comment:
Good cleanup.
I didn't see where 'interrupted' ever gets set

Reply:
This is a clone of the Parsedown code -- `interrupted` is set by Parsedown